src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -444,6 +457,9 @@ export default {
         // is this a paid filing?
         const isPaid = (header.status === FilingStatus.PAID)
 
+        // is this a completed filing?
+        const isCompleteIA = (header.status === FilingStatus.COMPLETED)
```

Comment:
parentheses required? Maybe a preference for readability? Either way, no action required, just curious :)

Reply:
Just followed isPaid, but I kind of like it? haha

Reply:
(FWIW - I prefer them - easier to see what this statement is doing.)